libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/report/PlainTextBuildReportWriter.kt

```diff
@@ -139,19 +159,18 @@ internal class PlainTextBuildReportWriter(
                 printMap(p, kind.name, attributesCounts.map { (k, v) -> k.readableString to v }.toMap())
             }
         }
-        p.println()
     }
 
     private fun printTaskOverview(build: BuildExecutionData) {
         var allTasksTimeMs = 0L
         var kotlinTotalTimeMs = 0L
-        val kotlinTasks = ArrayList<TaskExecutionData>()
+        val kotlinTasks = ArrayList<BuildOperationRecord>()
```

Comment:
Are these still only tasks, or we'll also get transforms here?

Reply:
It includes transforms too. All the naming in this class is still "tasks" because I wanted to keep this change small, and they're only private methods. Also, "build operations" is a new concept that people will have look it up to find out what it means. However, if you think it's more readable to rename "tasks" to "build operations" in this class, just let me know.

Reply:
Kdoc on BuildOperationRecord should provide enough info, hopefully. If it is not a big change, please include it in this PR. Otherwise, you can address it in another PR.

Reply:
Okay, I've added a note/TODO in this class.